src/django/api/facilities_download_view_set.py

```diff
@@ -84,3 +111,27 @@ def __can_user_download_over_limit(
             return False
 
         return True
+
+    @staticmethod
+    def reset_facility_download_limits_next_month(
```

Comment:
I don't think we really need this method, we could handle this logic by setting the default values.